cheatsheets/Access_Control_Cheat_Sheet.md

```diff
@@ -2,33 +2,33 @@
 
 Web applications need access controls to allow users (with varying privileges) to use the application. They also need administrators to manage the applications access control rules and the granting of permissions or entitlements to users and other entities. Various access control design methodologies are available. To choose the most appropriate one, a risk assessment needs to be performed to identify threats and vulnerabilities specific to your application, so that the proper access control methodology is appropriate for your application.
 
 ### Access Control Policy
 
 Why do we need an access control policy for web development?
 
-The intention of having an access control policy is to ensure that security requirements are described clearly to architects, designers, developers and support team, such that access control functionality is designed and implemented in a consistent manner.
+The intention of having an access control policy is to ensure that security requirements are described precisely to architects, designers, developers and support team, such that access control functionality is designed and implemented in a consistently.
 
 ## Role-Based Access Control (RBAC)
 
 In Role-Based Access Control (RBAC), access decisions are based on an individual's roles and responsibilities within the organization or user base.
 
-The process of defining roles is usually based on analyzing the fundamental goals and structure of an organization and is usually linked to the security policy. For instance, in a medical organization, the different roles of users may include those such as a doctor, nurse, attendant, patients, etc. Obviously, these members require different levels of access in order to perform their functions, but also the types of web transactions and their allowed context vary greatly depending on the security policy and any relevant regulations (HIPAA, Gramm-Leach-Bliley, etc.).
+The process of defining roles is usually based on analyzing the fundamental goals and structure of an organization and is usually linked to the security policy. For instance, in a medical organization, the different roles of users may include those such as a doctor, nurse, attendant, patients, etc. These members require different levels of access in order to perform their functions, but also the types of web transactions and their allowed context vary greatly depending on the security policy and any relevant regulations (HIPAA, Gramm-Leach-Bliley, etc.).
 
 An RBAC access control framework should provide web application security administrators with the ability to determine who can perform what actions, when, from where, in what order, and in some cases under what relational circumstances.
 
 The advantages of using this methodology are:
 
 - Roles are assigned based on organizational structure with emphasis on the organizational security policy
-- Easy to use
-- Easy to administer
+- Accessible to use
```

Comment:
This wording sounds strange strange, I have never heard anything described as "accessible to use" in this context. These points are talking about simplicity, not accessibility - so I'd revert this change and the one below.